utils/threshAlgos.py

```diff
@@ -1498,24 +1498,13 @@ def getDetName(entry):
                 for vfat in range(0,nVFATS):
                     canv_Summary1D.cd(chamber_vfatPos2PadIdx[gemType][vfat]).SetLogy()
 
-                    # make sure the inflection point is there
```

Comment:
I would probably keep the line drawing functionality, but disable it by default if someone looking at the plot finds it too distracting

Reply:
Okay, I have not added input options in python, so I will have to take a look at how to do this.

Reply:
Should I also disable by default the printing of the table? (The table will still be printed to the file)

Reply:
were there complaints about the table?
i think there are other tools that are printing tables to the output, so i wouldn't change that

Reply:
Okay, then I will default that option to true and leave it as an option if people complain about the table in the future :)